db/migrate/20211123060535_create_treatment_inertia_tables.rb

```diff
@@ -0,0 +1,35 @@
+class CreateTreatmentInertiaTables < ActiveRecord::Migration[5.2]
+  def up
+    create_table :clean_medicine_to_dosage, id: false do |t|
+      t.bigint :rxcui, null: false
+      t.string :medicine, null: false
+      t.float :dosage, null: false
+    end
+    execute "COPY clean_medicine_to_dosage FROM '#{Rails.root}/config/data/treatment-inertia/clean_medicine_to_dosage.csv' WITH DELIMITER ',' CSV HEADER"
+
+    create_table :raw_to_clean_medicine, id: false do |t|
+      t.string :raw_name, null: false
+      t.string :raw_dosage, null: false
+      t.bigint :rxcui, null: false
+    end
+    execute "COPY raw_to_clean_medicine FROM '#{Rails.root}/config/data/treatment-inertia/raw_to_clean_medicine.csv' WITH DELIMITER ',' CSV HEADER"
+
+    create_table :medicine_purpose, id: false do |t|
```

Comment:
Should we use the rxnorm codes to map medicines with purposes?

Reply:
Yeah, good idea. I also need to add the right kind of unique indexes on these. Will do this in the next pass. Thanks for this suggestion.

Reply:
This table has one row per medicine name. So, Amlo 5, 10, 20, etc are still just the one row. Which is nice, since this table is going to be maintained manually in the foreseeable future. I've added unique indexes on other tables with https://github.com/simpledotorg/simple-server/pull/3091.